src/Components/Components/src/CascadingValue.cs

```diff
@@ -167,12 +165,12 @@ void ICascadingValueComponent.Subscribe(ComponentState subscriber)
 
         void ICascadingValueComponent.Unsubscribe(ComponentState subscriber)
         {
-            _subscribers.Remove(subscriber);
+            _subscribers!.Remove(subscriber);
```

Comment:
```suggestion
            _subscribers?.Remove(subscriber);
```

Better safe than sorry?